MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/basics/KotlinSimpleMapViewActivity.kt

```diff
@@ -22,49 +24,50 @@ class SimpleMapViewActivityKotlin : AppCompatActivity() {
 
         // This contains the MapView in XML and needs to be called after the access token is configured.
         setContentView(R.layout.activity_basic_simple_kotlin)
+        mapView = findViewById(R.id.mapView)
```

Comment:
Why are we adding view inflation while we can use `import kotlinx.android.synthetic.main.activity_basic_simple_kotlin.*` instead?